app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdScreenLogicTest.kt

```diff
@@ -133,10 +137,15 @@ class ScanSimpleIdScreenLogicTest {
         moshi = Moshi.Builder().build()
     )
 
+    val features = Features(
+        remoteConfigService = NoOpRemoteConfigService(DefaultValueConfigReader()),
+        overrides = mapOf(Feature.IndiaNationalHealthID to true)
+    )
+
```

Comment:
Since we are not using `features` anymore, we can remove this right? You can do this in the next PR.